Source/Core/Core/State.h

```diff
@@ -81,13 +80,8 @@ struct StateExtendedHeader
 };
 
 void Init(Core::System& system);
-
 void Shutdown();
 
-void EnableCompression(bool compression);
-
-bool ReadHeader(const std::string& filename, StateHeader& header);
```

Comment:
Since `StateHeader` is no longer used in `State.h`, everything after the definition of `NUM_STATES` and before `Init` can be moved into `State.cpp`.